mesonbuild/compilers.py

```diff
@@ -184,10 +184,10 @@ def is_library(fname):
                 'winspool.lib', 'shell32.lib', 'ole32.lib', 'oleaut32.lib',
                 'uuid.lib', 'comdlg32.lib', 'advapi32.lib']
 
-gnu_color_args = {'auto' : ['-fdiagnostics-color=auto'],
+gnu_color_args = {'auto': ['-fdiagnostics-color=auto'],
                   'always': ['-fdiagnostics-color=always'],
-                  'never' : ['-fdiagnostics-color=never'],
-                  }
+                  'never': ['-fdiagnostics-color=never'],
+                 }
```

Comment:
__[flake8]__
* [E124] closing bracket does not match visual indentation

<!-- Comment by [SideCI](https://sideci.com) -->